Change default vote weight to 51

branders0n opened the pull request:
Stepping towards adding this to settings page. Let's try this first!

sktbrd commented:
I think the voting solution looks good, but we dont need the surpressHydrationWarning unless is a life or death case. Can you remove those, I can take care of the hydration errors in another PR. 

Gonna deploy on vercel so I can test the voting functionality now. Thanks a lot for contributing man, its really really dope.

branders0n commented:
@sktbrd ty bro I am trying but it is still giving me some grief. Did this work? I am really wanting to keep at it and get this confirmed that my repo and PR submissions all come through OK. This is my first time using github so maybe we need to chat more to make sure

Let me know how testing goes, I reverted the hydration error suppression. Only reason I added that was because of the errors I got in dev environment, but maybe it doesn't show up for real users

sktbrd commented:
@branders0n you are doing great. the build error we were having came from web-gnars branch, a tricky typeerror on his complex feature, he is also killing it, but stuff like this happens, I just fixed and submitted to main, now I got you pr locally merged main and I am trying to commit to your PR so we can preview the vote functionallity
